python_src/src/lamp_py/performance_manager/l0_gtfs_rt_events.py

```diff
@@ -123,18 +125,9 @@ def combine_events(
     return events
 
 
-def compute_metrics(events: pandas.DataFrame) -> pandas.DataFrame:
-    """
-    generate dwell times, stop times, and headways metrics for events
-    """
-    # TODO(zap / ryan) - figure out how 2 compute these. the update
-    # logic will also need to be updated.
-    return events
-
-
 def upload_to_database(
     events: pandas.DataFrame, db_manager: DatabaseManager
-) -> None:
+) -> pandas.DataFrame:
```

Comment:
Return the events from `upload_to_database` for use in the trips / metrics loading pipeline.